src/service/url-replacements-impl.js

```diff
@@ -256,7 +256,8 @@ export class GlobalVariableSource extends VariableSource {
 
         // A temporary work around to extract Client ID from _ga cookie. #5761
         // TODO: replace with "filter" when it's in place. #2198
-        if (scope == '_ga') {
+        const cookieName = opt_cookieName || scope;
```

Comment:
well, it's not worth it to introduce "viewer" checking logic here. This is a no-op if it does not see GA formatted string.

as you said, it's really a temporary hack.